kafka-client/src/main/java/dev/responsive/kafka/api/async/internals/AsyncThreadPool.java

```diff
@@ -110,6 +110,10 @@ public void removeProcessor(final String processorName, final int partition) {
     if (inFlightForTask != null) {
       log.info("Cancelling {} pending records for {}[{}]",
                inFlightForTask.size(), processorName, partition);
+      if (!inFlightForTask.isEmpty()) {
+        log.info("ANTITHESIS SOMETIMES: cancelling {} pending records for {}[{}]",
```

Comment:
antithesis looks for ANTITHESIS SOMETIMES messages to make sure they are called at least once on each test run